deploy/porter/base/kustomization.yaml

```diff
@@ -10,5 +10,5 @@ generatorOptions:
 
 images:
   - name: manager
-    newName: ghcr.io/getporter/porter-operator-manager
-    newTag: v0.8.2-28-g37ed573
+    newName: ghcr.io/troy0820/porter-operator
```

Comment:
All looks good. I would just expect this to be coming from the getporter group.  I've been out of the loop (but coming back).  So just let me know that this is intentional.

Reply:
I didn't have rights yet but I can change this. It's there now.